src/read_write/ply.rs

```diff
@@ -222,25 +223,25 @@ fn parse_header<R: BufRead>(reader: &mut R) -> Result<(Header, usize)> {
     ))
 }
 
-type ReadingFn = fn(nread: &mut usize, buf: &[u8], val: &mut Point);
+type ReadingFn = fn(nread: &mut usize, read_buf: &[u8], write_buf: &mut [u8], write_pos: usize);
 
 // The two macros create a 'ReadingFn' that reads a value of '$data_type' out of a reader, and
```

Comment:
Is this comment still up to date? There is no `$data_type` in the code.

Reply:
I changed it back to `$data_type`